src/BenchmarkTools.jl

```diff
@@ -12,7 +12,7 @@ using Profile
 const BENCHMARKTOOLS_VERSION = if VERSION >= v"1.9"
     pkgversion(BenchmarkTools)
 else
-    v"1.4.0"
```

Comment:
What is this line doing? is `pkgversion` supported on 1.6 and above? if so, perhaps we can simplify this to just call `pkgversion`.